flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
libxml requires rust-bindgen during build as of 0.3.4

A workaround is to export `LIBXML2=/path/to/libxml2.so`. But I just found out the path differs depending on  the architecture. And I don't know how to export different variables for builds on x64 and aarch64.

Reply:
and you can have per architecture exports too.

Reply:
That would be nice. Can you point me to some docs?